backtrace.h

```diff
@@ -7,13 +7,13 @@ modification, are permitted provided that the following conditions are
 met:
 
     (1) Redistributions of source code must retain the above copyright
-    notice, this list of conditions and the following disclaimer. 
```

Comment:
Thanks, but the files like backtrace.h and so forth are simply copied from the libbacktrace library in GCC.  I don't particularly want to change them, except perhaps to update them with a new copy from GCC.  If that is done it should be in a change by itself with no other changes.